presto-main/src/main/java/com/facebook/presto/execution/CreateRoleTask.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
missing tests... maybe they are added later once role support is added to hive connector